app/main.ipynb

```diff
@@ -31,17 +35,777 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code for task 0 here"
+    "adult = pd.DataFrame(pd.read_csv(\"../data/adult.csv\"))\n",
+    "adult.describe()"
    ],
```

Comment:
you should check data for correctness. So if salary = "<=50K", then you should ensure that column "salary K$" has an appropriate value